pycaret/containers/models/time_series.py

```diff
@@ -856,6 +856,69 @@ def _set_tune_distributions(self) -> Dict[str, List[Any]]:
             }
         return tune_distributions
 
+class CrostonContainer(TimeSeriesContainer):
+    model_type = TSModelTypes.CLASSICAL
+
+    def __init__(self, globals_dict: dict) -> None:
+        logger = get_logger()
+        np.random.seed(globals_dict["seed"])
+        self.gpu_imported = False
+
+        from sktime.forecasting.croston import Croston  # type: ignore
+
+        dummy = Croston()
+        self.active:bool = self.disable_pred_int_enforcement(
+            forecaster=dummy, enforce_pi=globals_dict["enforce_pi"]
+        )
+        if not self.active:
+            return
+
+        # self.seasonality_present = globals_dict.get("seasonality_present")
+        # TODO: Not sure if this is required.
+        sp = globals_dict.get("seasonal_period")
+        self.sp = sp if sp is not None else 1
+
+        self.strictly_positive = globals_dict.get("strictly_positive")
+
+        args = self._set_args
+        tune_args = self._set_tune_args
+        tune_grid = self._set_tune_grid
+        tune_distributions = self._set_tune_distributions
+        leftover_parameters_to_categorical_distributions(tune_grid, tune_distributions)
+
+        eq_function = lambda x: type(x) is Croston
+
+        super().__init__(
+            id="croston",
+            name="Croston",
+            class_def=Croston,
+            args=args,
+            tune_grid=tune_grid,
+            tune_distribution=tune_distributions,
+            tune_args=tune_args,
+            is_gpu_enabled=self.gpu_imported,
+            eq_function=eq_function,  # Added to differentiate between ExponentialSmoothing and Theta which are of same parent class
+        )
+
+    @property
+    def _set_args(self) -> Dict[str, Any]:
+        args = {"smoothing" : 0.1} # just use the default : smoothing = 0.1 
```

Comment:
I would not hard code it here. Leave it up to `sktime` to handle that. They anyway set the defaults.

Reply:
So in this case I would just leave an empty dict @ngupta23 ?

Reply:
You can just leave out the class property `_set_args` (since it inherits from the base class). See example here: https://github.com/pycaret/pycaret/blob/7f302caad7841c997c4ea8653a39eff7da56e681/pycaret/containers/models/time_series.py#L443

Reply:
thanks, I assume it doesnt need to be passed to super/base class constructor?

Reply:
Nevermind, I have worked this out. Removed the code